daemon/src/projection.rs

```diff
@@ -855,6 +858,21 @@ impl Actor {
             .send_cfds_update(self.state.cfds.clone(), self.state.quote);
     }
 
+    /// Updates the projection state with the new CFD.
+    ///
+    /// In theory, we could send freshly minted CFD over but it is a different Rust struct and
+    /// mapping it is tedious. Loading a new CFD from the DB is cheap so we just do that
+    /// instead.
+    async fn handle(&mut self, msg: NewCfd) {
```

Comment:
Maybe we should do it nevertheless? 
Loading a cfd means executing 2 queries sequentially, which also involves loading and applying the events which is not "that" cheap imho. 
Mapping it in code should give us an additional performance boost.

Reply:
But a new CFD has 0 events! That is what made me in the end go with re-loading because it is essentially just a query to the `cfds` table.